static/app/utils/reactRouter6Compat/router.tsx

```diff
@@ -75,6 +76,22 @@ interface RedirectProps extends Omit<NavigateProps, 'to'> {
  */
 function Redirect({to, ...rest}: RedirectProps) {
   const params = useParams();
+  const routes = useRoutes();
+
+  // Capture sentry error for this redirect. This will help us understand if we
+  // have redirects that are unused or used too much.
+  useEffect(() => {
+    const routePath = routes
+      .map(route => route.path ?? '')
+      .filter(path => path !== '')
+      .join('/');
+
+    Sentry.captureMessage('Redirect route used', {
+      level: 'info',
+      tags: {routePath},
+      fingerprint: [routePath],
```

Comment:
Do you want one error per route? Or have a single error with route as a tag?

Reply:
hmm. Maybe it's OK to just have one, and we can use discover to drill down